src/components/publicbeat/PublicBeat.tsx

```diff
@@ -165,7 +165,7 @@ const PublicBeat: React.FC<PublicBeatProps> = ({...props}) => {
 
     axios.post(url + '/api/user/like_vertex', data, config)
     .then((res) => {
-      console.log(res);
+      // console.log(res);
```

Comment:
Do we need to keep commented out console logs here or can we remove them entirely?

Reply:
We can just remove it yea, maybe in real big app, it will send to a production team to handle the err I guess